src/hooks/protections/can_apply_data.cpp

```diff
@@ -1318,6 +1318,18 @@ namespace big
 					notify::crash_blocked(sender, "invalid object model");
 					return true;
 				}
+				//auto objPos = creation_node->m_dummy_position; // With YimMenu this is odly 0,0,0 ...
+				//auto myPos  = g_local_player->get_position();
```

Comment:
Remove any commented out code unless there's a clear explanation written above/around/under it.

This should be looked into though.